src/applications/personalization/profile-2/components/connected-apps/components/ConnectedApps.jsx

```diff
@@ -0,0 +1,130 @@
+  };
+
+  render() {
+    const { accounts } = this.props;
+    const deletedAccounts = accounts
+      ? accounts.filter(account => account.deleted)
+      : [];
+    const activeAccounts = accounts
+      ? accounts.filter(account => !account.deleted)
+      : [];
+
+    return (
+      <div className="va-connected-acct">
+        <div className="">
+          {accounts && (
+            <p className="va-introtext vads-u-font-size--md">
+              You've given these third-party apps or websites access to some of
+              your Veteran data, like health or service records. You can remove
+              their access at any time by disconnecting the app. Disconnected
+              apps can't receive any new data from VA, but may still have access
+              to information that you've previously shared.
+            </p>
+          )}
+
+          {!accounts && (
+            <p className="va-introtext vads-u-font-size--md">
+              You don’t currently have any third-party apps or websites
+              connected to your Veteran data, like health or service records.
+              When you do, this is where you can manage them.
+            </p>
+          )}
+
+          {deletedAccounts.map(account => (
+            <AppDeletedAlert
+              id={account.id}
+              accountName={account?.attributes?.title}
+              key={account.id}
+              dismissAlert={this.dismissAlert}
+            />
+          ))}
+
+          {activeAccounts.map((account, idx) => (
+            <ConnectedApp
+              key={account.id}
+              confirmDelete={this.confirmDelete}
+              isLast={idx + 1 === activeAccounts.length}
+              {...account}
+            />
+          ))}
+
+          <div className="vads-u-display--flex vads-u-flex-direction--column vads-u-background-color--gray-lightest vads-u-padding--2p5 vads-u-margin-top--2">
+            <h3 className="vads-u-margin--0">
+              Have questions about connecting to VA.gov?
+            </h3>
+            <p>
+              Get answers to frequently asked questions about how connected
+              third-party apps work, what types of information they can see, and
+              the benefits of sharing your information.
+            </p>
+
+            <a
+              className="vads-u-color--primary-alt-darkest"
+              href="/sign-in-faq/"
+              onClick={() =>
+                recordEvent({
+                  event: 'account-navigation',
+                  'account-action': 'view-link',
+                  'account-section': 'vets-faqs',
+                })
+              }
+            >
+              Go to Connected Account FAQs
+            </a>
+
+            <a
+              className="vads-u-color--primary-alt-darkest"
+              href="/third-party-applications/"
+              onClick={() =>
+                recordEvent({
+                  event: 'find-third-party-applications',
```

Comment:
is this an existing event we are reporting to GA?

Reply:
It is not, the page it links to does not yet exist and is I believe being built by public websites. I'll make sure that actually the case.